docs/source/postprocessing/openPMD.rst

```diff
@@ -0,0 +1,6 @@
+.. _pp-openPMD:
+
+openPMD
+=======
+
+Please see https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Post-processing-and-Visualization for now.
```

Comment:
QUESTION:
is the link correct or did you want to link to the openPMD repo/website?